assets/scss/components/main/_separators.scss

```diff
@@ -8,7 +8,7 @@
 	}
 
 	&.is-style-dots {
-		border-top: 4px dotted;
+		border-top: 4px dotted !important;
```

Comment:
What would happen if a plugin would want to add its own styles to this separator? Or if we require to change how this looks from the Otter or Neve Pro?